.github/workflows/workflow.yaml

```diff
@@ -0,0 +1,31 @@
+name: "Prow github actions"
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  execute:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: jpmcb/prow-github-actions@v1.1.3
+        with:
+          prow-commands: '/assign 
```

Comment:
We should probably evaluate which of these commands we want to support in kyverno and the additional set of privileges it provides to users who are not collaborators or maintainers. The /kind, /milestone and /assign and /unassign seem safer to enable for now. We can always add more over time if needed.